app/controllers/application_controller.rb

```diff
@@ -3,5 +3,6 @@ class ApplicationController < ActionController::Base
 
   include SessionsHelper
   include UsersHelper
+  include ApplicationHelper
```

Comment:
Aren't these helper methods already inherited by all children of ApplicationController?